neo/rawio/neuralynxrawio/neuralynxrawio.py

```diff
@@ -128,32 +131,58 @@ class NeuralynxRawIO(BaseRawIO):
         ("channel_id", "uint32"),
         ("sample_rate", "uint32"),
         ("nb_valid", "uint32"),
-        ("samples", "int16", (NcsSection._RECORD_SIZE)),
+        ("samples", "int16", NcsSection._RECORD_SIZE),
     ]
 
     def __init__(
-        self, dirname="", filename="", exclude_filename=None, keep_original_times=False, strict_gap_mode=True, **kargs
+            self,
+            dirname="",
+            include_filenames=None,
+            exclude_filenames=None,
+            keep_original_times=False,
+            strict_gap_mode=True,
+            **kargs
     ):
 
-        if dirname != "":
-            self.dirname = dirname
-            self.rawmode = "one-dir"
-        elif filename != "":
-            self.filename = filename
-            self.rawmode = "one-file"
+        if include_filenames is None:
```

Comment:
here we would check that filename is None. If it is not none we raise an error and mention the deprecation again, so that old users know how to fix for the API change.

Reply:
Shall we just raise a deprecation warning and assign `filename` to `include_filenames`? And do similar for `execlude_filename`?

Reply:
Yeah you could do that too!